### PR TITLE
Launch karma tests in browser

### PR DESCRIPTION
**Pull request by magarnier**

### CODE REVIEW

File: languages/en/developer-guide/front-end/karma.rst [hunk 1]
```diff
@@ -83,6 +83,39 @@ To run your tests locally, use the :ref:`npm scripts <npm_scripts>`.
 
     - Chromium is installed on your machine if it runs under Linux.
     - Chrome is installed otherwise.
+    
+How to debug tests
+^^^^^^^^^^^^^^^^^^
+
+To ease test debugg, you should launch them directly in your browser, to do this:
```
Comment (gorkat) [inline languages/en/developer-guide/front-end/karma.rst]: "To ease test debugging"

Comment (gorkat) [inline languages/en/developer-guide/front-end/karma.rst]: "you can",  people do not have to do it each time they run the Karma tests

File: languages/en/developer-guide/front-end/karma.rst [hunk 2]
```diff
@@ -83,6 +83,39 @@ To run your tests locally, use the :ref:`npm scripts <npm_scripts>`.
...
+     ...
+    browsers  : [
+                process.platform !== 'linux'
+                    ? 'ChromeHeadless'
+                    : 'Chromium'
+            ]
+    ...
+    }
+
+Launch `npm run watch` on your app folder, and you should see that all tests are executed in browser.
```
Comment (gorkat) [inline languages/en/developer-guide/front-end/karma.rst]: May I suggest to explain that we have to click on "debug" then open the web-console to see the errors?

File: languages/en/developer-guide/front-end/karma.rst [hunk 3]
```diff
@@ -83,6 +83,39 @@ To run your tests locally, use the :ref:`npm scripts <npm_scripts>`.
...
+^^^^^^^^^^^^^^^^^^
+
+To ease test debugg, you should launch them directly in your browser, to do this:
+
+In `/tools/utils/script/karma-configurator.js`, update `singleRun` parameter from `true` to `false` 
+
+.. code-block:: Javascript
+
+    function configureKarmaForSingleTest() {
+        return {
```
Comment (gorkat) [inline languages/en/developer-guide/front-end/karma.rst]: nope, this is for "npm run test", no need to modify this, only setting Chromium as browser is enough.

File: languages/en/developer-guide/front-end/karma.rst [hunk 4]
```diff
@@ -83,6 +83,39 @@ To run your tests locally, use the :ref:`npm scripts <npm_scripts>`.
...
+            singleRun: false,
+    ...
+    }
+
+
+In `/tools/utils/script/karma-common-config.js` update browser form `ChromiumHeadless` to `Chromium`
```
Comment (gorkat) [inline languages/en/developer-guide/front-end/karma.rst]: set the value of the  `browsers` property to `Chromium`

**Comment by Hyzual** [comment]
LGTM

**Pull request merged**